sormas-backend/src/main/java/de/symeda/sormas/backend/util/PatchHelper.java

```diff
@@ -0,0 +1,311 @@
+	 * 
+	 * @param jsonObject
+	 *            - the json payload containing the fields that are to be replaced. The structure of the json matches the structure of the
+	 *            Dto given by the existingObject
+	 * @param existingObject
+	 *            - the existing object whose fields are to be updated
+	 * @param <T>
+	 *            - the type of the object
+	 */
+	@Experimental
+	public static <T extends HasUuid> void postUpdate(@NotNull JsonNode jsonObject, @NotNull T existingObject) {
+
+		if (!existingObject.getUuid().equals(jsonObject.get(EntityDto.UUID).textValue())) {
+			throw new RuntimeException("The updated object must have a Uuid");
+		}
+		postUpdateOrReplace(jsonObject, existingObject);
+
+	}
+
+	private static <T extends HasUuid> void postUpdateOrReplace(@NotNull JsonNode jsonObject, @NotNull T existingObject) {
+		try {
+
+			Iterator<Map.Entry<String, JsonNode>> jsonObjectFieldMap = jsonObject.fields();
+			while (jsonObjectFieldMap.hasNext()) {
+				try {
+					Map.Entry<String, JsonNode> jsonObjectFieldMapEntry = jsonObjectFieldMap.next();
+					JsonNode jsonObjectFieldNode = jsonObjectFieldMapEntry.getValue();
+					Field existingObjectField = getFieldByName(jsonObjectFieldMapEntry.getKey(), existingObject.getClass());
+					if (existingObjectField == null) {
+						throw new RuntimeException("No such field exception! " + jsonObjectFieldMapEntry.getKey());
+					}
+					existingObjectField.setAccessible(true);
+
+					if (jsonObjectFieldNode.isObject()) {
+						updateExistingObject(existingObject, jsonObjectFieldNode, existingObjectField);
+
+					} else if (jsonObjectFieldNode.isArray()) {
+						updateObjectList(existingObject, jsonObjectFieldNode, existingObjectField);
+
+					} else {
+						setFieldValue(existingObject, existingObjectField, jsonObjectFieldNode);
+					}
+
+				} catch (IllegalAccessException e) {
+					e.printStackTrace();
+					Logger logger = LoggerFactory.getLogger(existingObject.getClass());
+					logger.error(String.format("Failed to update object" + existingObject.getClass().getSimpleName() + ". Error: " + e.getMessage()));
+				}
+
+			}
+
+		} catch (Exception e) {
+			e.printStackTrace();
+			throw new RuntimeException(e);
+		}
+	}
+
+	/**
+	 * This method updates the fields of type collections (lists or array).
+	 * 
+	 * @param existingObject
+	 * @param jsonObjectFieldNode
+	 * @param existingObjectField
+	 * @param <T>
+	 * @throws JsonProcessingException
+	 * @throws IllegalAccessException
+	 */
+	private static <T extends HasUuid> void updateObjectList(T existingObject, JsonNode jsonObjectFieldNode, Field existingObjectField)
+		throws JsonProcessingException, IllegalAccessException {
+		Class listElementClass = getParameterizedType(existingObjectField);
+		ArrayList tempNewElementList = new ArrayList();
+
+		for (JsonNode listElement : jsonObjectFieldNode) {
+			addOrReplaceListElement(existingObject, existingObjectField, listElementClass, tempNewElementList, listElement);
```

Comment:
I don't really like the procedural approach where you start with en empty list and calling a method fills it, it's hard to find and debug where and how the items get added into the list.

Would be cleaner to have a "merge" method that using the existing list and the json list creates a new list that can be set as value on the dto